src/axom/lumberjack/Message.cpp

```diff
@@ -266,7 +269,11 @@ const char* packMessages(const std::vector<Message*>& messages)
 {
   if(messages.size() == 0)
   {
-    return zeroMessage;
+    // Allocate space for zero and null character
+    char* zeroString = new char[zeroMessage().size() + 1];
+    std::memcpy(zeroString, zeroMessage().c_str(), zeroMessage().size());
+    zeroString[zeroMessage().size()] = '\0';
+    return zeroString;
```

Comment:
@bmhan12 -- looks like this will leak memory each time it's called.

Can `zeroMessage == "0"` be a class member of Message?  (or even better, a static class member?)

Reply:
Also, we probably don't want an allocation each time this is called with no messages.